src/components/Navbar/Navbar.component.jsx

```diff
@@ -14,35 +14,38 @@ import { useStyles } from './Navbar.style';
 import { NavbarSearch } from './NavbarSearch/NavbarSearch.component';
 import { SectionDesktop } from './SectionDesktop/SectionDesktop.component';
 import { SectionMobile } from './SectionMobile/SectionMobile.component';
+import { Link } from 'react-router-dom';
 
 const Navbar = () => {
     const [isOpen, setIsOpen] = useState(false);
-    const { menuButton, grow, title } = useStyles();
+    const { menuButton, grow, title, parent } = useStyles();
 
     return (
-        <div className={grow}>
-        <AppBar position="static">
-            <Toolbar>
-                <IconButton
-                     edge="start" 
-                     className={menuButton} 
-                     color="inherit" 
-                     aria-label="menu"
-                     onClick={ () => setIsOpen(!isOpen) }
-                >
-                    <MenuIcon/>
-                </IconButton>
-                <Typography variant="h6" className={title}>
-                    YoutubePerron
-                </Typography>
-                <div className={grow} />
-                <NavbarSearch/>
-                <div className={grow} />
-                <SectionDesktop/>
-                <SectionMobile/>
-            </Toolbar>
-        </AppBar>
-        <Menu isOpen={isOpen} setIsOpen={setIsOpen} />
+        <div className={ parent }>
+            <AppBar position="fixed">
```

Comment:
Desconozco si sea sencillo implementar https://wicky.nillia.ms/headroom.js/
pa que se vea al millón el nav

Reply:
```suggestion
            <AppBar position="static">
```
:v ya con eso se quita de arriba con el scroll o hace algo mas en especial eso?